packages/at_secondary_server/lib/src/verb/handler/keys_verb_handler.dart

```diff
@@ -0,0 +1,195 @@
+import 'dart:collection';
+import 'dart:convert';
+
+import 'package:at_commons/at_commons.dart';
+import 'package:at_persistence_secondary_server/at_persistence_secondary_server.dart';
+import 'package:at_secondary/src/connection/inbound/inbound_connection_metadata.dart';
+import 'package:at_secondary/src/constants/enroll_constants.dart';
+import 'package:at_secondary/src/enroll/enroll_datastore_value.dart';
+import 'package:at_secondary/src/server/at_secondary_impl.dart';
+import 'package:at_secondary/src/verb/handler/abstract_verb_handler.dart';
+import 'package:at_server_spec/at_server_spec.dart';
+import 'package:at_server_spec/at_verb_spec.dart';
+
+class KeysVerbHandler extends AbstractVerbHandler {
+  static Keys keys = Keys();
+
+  KeysVerbHandler(SecondaryKeyStore keyStore) : super(keyStore);
+
+  // Method to verify whether command is accepted or not
+  // Input: command
+  @override
+  bool accept(String command) => command.startsWith('keys:');
+
+  // Method to return Instance of verb belongs to this VerbHandler
+  @override
+  Verb getVerb() {
+    return keys;
+  }
+
+  // Method which will process update Verb
+  // This will process given verb and write response to response object
+  // Input : Response, verbParams, AtConnection
+  @override
```

Comment:
The method comment above is useless. Please either remove it or replace it with something more useful

Reply:
Removed.
https://github.com/atsign-foundation/at_server/pull/1496/commits/b22ea2028164aa54299758080d3b9033bf3a98e2